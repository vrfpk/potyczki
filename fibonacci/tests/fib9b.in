21832689027670219
