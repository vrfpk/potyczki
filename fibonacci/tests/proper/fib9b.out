182049362199459112
